examples/webgl_gpgpu_birds.html

```diff
@@ -513,42 +444,86 @@
 				velocityVariable.wrapT = THREE.RepeatWrapping;
 				positionVariable.wrapS = THREE.RepeatWrapping;
 				positionVariable.wrapT = THREE.RepeatWrapping;
-
+				
 				var error = gpuCompute.init();
 				if ( error !== null ) {
-
 				    console.error( error );
-
 				}
-
 			}
 
 			function initBirds() {
 
-				var geometry = new BirdGeometry();
-
-				// For Vertex and Fragment
-				birdUniforms = {
-					"color": { value: new THREE.Color( 0xff2200 ) },
-					"texturePosition": { value: null },
-					"textureVelocity": { value: null },
-					"time": { value: 1.0 },
-					"delta": { value: 0.0 }
-				};
-
-				// THREE.ShaderMaterial
-				var material = new THREE.ShaderMaterial( {
-					uniforms: birdUniforms,
-					vertexShader: document.getElementById( 'birdVS' ).textContent,
-					fragmentShader: document.getElementById( 'birdFS' ).textContent,
-					side: THREE.DoubleSide
-
-				} );
-
-				var birdMesh = new THREE.Mesh( geometry, material );
+				var geometry = BirdGeometry;
+
+				var m = new THREE.MeshStandardMaterial({
+					vertexColors : true,
+					flatShading : true,
+					roughness : 1,
+					metalness : 0
+				})
+
+				m.onBeforeCompile = ( shader ) => {
+
+					shader.uniforms.texturePosition = { value : null };
+					shader.uniforms.textureVelocity = { value : null };
+					shader.uniforms.textureAnimation = { value : textureAnimation };
+					shader.uniforms.time = { value : 1.0 };
+					shader.uniforms.size = { value : 0.1 };
+					shader.uniforms.delta = { value : 0.0 };
+
+					var token = '#define STANDARD';
+        			var insert = 'attribute vec4 reference;';
+					insert += 'attribute vec4 seeds;';
```

Comment:
This section would be nicer written as a template string. @Mugen87 do we allow these in examples yet?

Reply:
Yes, I think we can use template strings in examples. They're using modules anyway.

Reply:
@santi-grau can you change this to use template strings?

``` glsl
var insert = /* glsl */`
    attribute vec4 reference;
    attribute vec4 seeds;
    attribute vec3 birdColor;
    uniform sampler2D texturePosition;
    ...  
    `
```

Reply:
Sure! will look into all this during the weekend :)

Reply:
Put all the 'replacement' strings into a template string. Hope this is what you meant. :S